### PR TITLE
Push all packages in a project to central

### PR DESCRIPTION
**Pull request by NatashaWso2**
```bash
ballerina push
ballerina install or ballerina push --repository home
```

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java [hunk 1]
```diff
@@ -64,6 +66,7 @@
     private static final Path BALLERINA_HOME_PATH = RepoUtils.createAndGetHomeReposPath();
     private static final Path SETTINGS_TOML_FILE_PATH = BALLERINA_HOME_PATH.resolve(
             ProjectDirConstants.SETTINGS_FILE_NAME);
+    private static Manifest manifest = new Manifest();
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java]: We can make this local to both functions right ? "pushPackages" & "pushAllPackages"

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java [hunk 2]
```diff
@@ -338,4 +341,43 @@ private static String readSummary(String mdFileContent) {
...
+                                         .filter(path -> Files.isDirectory(path, LinkOption.NOFOLLOW_LINKS))
+                                         .map(ProjectDirs::getLastComp)
+                                         .filter(dirName -> !isSpecialDirectory(dirName))
+                                         .map(Path::toString).collect(Collectors.toList());
+            if (fileList.size() == 0) {
+                throw new BLangCompilerException("no packages found to push in " + sourceRootPath.toString());
+            }
+            fileList.forEach(path -> pushPackages(path, sourceRoot, home));
+        } catch (IOException ex) {
+            throw new BLangCompilerException("error occured when pushing packages from " + sourceRootPath.toString(),
```
Comment (anupama-pathirage) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java]: typo in occured?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java (hunk 1 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java]: Fixed with 8534f02

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java (hunk 2 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java]: Fixed with 0cabb9a

**Pull request merged**